src/wp-admin/css/edit.css

```diff
@@ -1415,7 +1415,7 @@ p.description code {
 
 p.popular-tags {
 	border: none;
-	line-height: 2em;
+	line-height: 2;
```

Comment:
```suggestion
	line-height: 2em; /* keep the em unit because tag links use multiple font sizes */
```
---
The tag cloud for **Featured** plugins available to install apparently needs the `em` unit.

Before patch (using WordPress 6.4.2, without editing browser font size):

![line height is 2em](https://github.com/WordPress/wordpress-develop/assets/17100257/0036394b-8930-4fea-a480-15cb22c09727)

With PR5719 (still standard font sizes)

![line height is taller](https://github.com/WordPress/wordpress-develop/assets/17100257/f611358a-6c58-4a74-91d9-2e024f48fd40)